### PR TITLE
App Settings Cleanup i

### PR DESCRIPTION
**Pull request by mindgraffiti**
Ref #11925
Ref https://github.com/wordpress-mobile/WordPress-iOS/pull/18569

## Summary
Part 1 of this PR updates some of the UI in App Settings to follow our current color scheme and improves the UX for clearing the Media Cache. This PR does not address the changes to the "Max Image Upload Size" row or the new location option. It is arriving in a separate future PR.

**Changes include:**
- Clear Spotlight Index button uses blue brand color and is left aligned
- Clear Siri Shortcut Suggestions uses blue brand color and is left aligned
- Media Cache Size UI is updated to show a disclosure arrow and navigates to its own ViewController 
- "Other" section header remains intact because more options are listed in that section compared to when the ticket was created

## Screenshots
| Before | After |
| --- | --- |
| Simulator Screen Shot - iPhone 12 Pro - 2022-05-04 at 14 20 06 | Simulator Screen Shot - iPhone 12 Pro - 2022-05-04 at 16 04 50 |

## To test:
0. Install the app on a device, with iOS 12 or higher. Steps 7 and 8 are features found only in iOS 12 and up.
1. Select "My Site" in the bottom nav baar
2. Select your profile picture
3. Choose "App Settings" in the modal view
4. The "Media Cache Size" row should display the current size of the media cache
5. Select the row. It should navigate to a new ViewController with the "Media Cache Size" title and size row, as well as a row button to clear the cache. Clear the cache and verify the size reports "Empty"
6. Navigate back to App Settings
7. Tap the "Clear Spotlight Index" and wait for a toast message that it was successful
8. Tap the "Clear Siri Shortcut Suggestions" and wait for a toast message that it was successful 

## Regression Notes
1. Potential unintended areas of impact
- App Settings
- WPImmuTableRows contains a new row button style

6. What I did to test those areas of impact (or what existing automated tests I relied on)
Run all unit tests, which includes:
- ImmuTableTest 
- MediaFileManagerTests 

10. What automated tests I added (or what prevented me from doing so)
- None, UI tests don't seem like a valuable use of time

PR submission checklist:
- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@
 -----
 
 
+19.9
+-----
+* [*] App Settings: refreshed the UI with updated colors for Media Cache Size controls, Clear Spot Index row button, and Clear Siri Shortcut Suggestions row button. From destructive (red color) to standard and brand colors.
+
```
Comment (Gio2018) [inline RELEASE-NOTES.txt]: Since we are now targeting `20.0`, the release notes should be updated as well.

**Review by Gio2018 — CHANGES_REQUESTED** [review]
This works as expected @mindgraffiti ! I noticed though that the release notes need to be updated to the next release.
Thank you!

**Comment by Gio2018** [comment]
Hey @mindgraffiti , can you please add the PR reference in the release notes, like the entry above yours? Thank you!

**Comment by mindgraffiti** [comment]
@Gio2018 done! 🙂

**Review by Gio2018 — APPROVED** [review]
:shipit:

**Pull request merged**